admin/README.md

```diff
@@ -0,0 +1,3 @@
+# Administration UI
+
+This user interface allows partners to manage their Business Messaging agents. Visit the [developer site](https://developers.google.com/business-communications/business-messages) to learn more.
```

Comment:
"This user interface allows you to manage your Business Messages agent."

Reply:
fixed